tensorboard/backend/event_processing/data_ingester.py

```diff
@@ -273,4 +273,4 @@ def _check_filesystem_support(paths):
                     + " filesystem support, consider installing TensorFlow I/O"
                     + " (https://www.tensorflow.org/io) via `pip install tensorflow-io`."
                 ).format(missing_scheme, supported_schemes_msg),
-            )
+            ) from None
```

Comment:
typo?

Reply:
Ah never mind I misread this and didn't realize it was attached to the `raise` (it's hidden in the GH diff).

That said, you might actually want to raise it `from e` where `e` is the original ImportError, since it shows the full trace then but with a clearer message ("The above exception was the direct cause of the following exception:" versus "During handling...").  Up to you though.

Reply:
Done.